.python-version

```diff
@@ -0,0 +1 @@
+3.12.8
```

Comment:
Can we allow multiple python version? Like all 3.12.x as they should be backwards compatible. This would save some space for some users that have slightly different version of python. For me ubuntu 24.10 have python 3.12.7 so it would be nice if I could reuse it instead of installing new version.

Per https://docs.astral.sh/uv/concepts/python-versions/#requesting-a-version this should be possible.

Reply:
Good point. I've done this in [61b521e](https://github.com/servo/servo/pull/34504/commits/61b521efc435a0bc566598149fcacad2b4b67d3a)